zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/UserTaskControllerTest.java

```diff
@@ -432,6 +439,98 @@ public void shouldYieldBadRequestWhenUpdateTaskWithoutActionAndChanges() {
     Mockito.verifyNoInteractions(brokerClient);
   }
 
+  @Test
+  public void shouldYieldBadRequestWhenUpdateTaskWithoutMalformedDueDate() {
+    // given
+    final var request = new UserTaskUpdateRequest().changeset(new Changeset().dueDate("foo"));
+
+    final var expectedBody =
+        ProblemDetail.forStatusAndDetail(
+            HttpStatus.BAD_REQUEST,
+            "The provided due date 'foo' cannot be parsed as a date according to RFC 3339, section 5.6.");
+    expectedBody.setTitle("INVALID_ARGUMENT");
+    expectedBody.setInstance(URI.create("/api/v1/user-tasks/1"));
+
+    // when / then
+    webClient
+        .patch()
+        .uri("api/v1/user-tasks/1")
```

Comment:
:x: We updated the REST API base path, it's just `/v1` now instead of `api/v1`. 

:upside_down_face: We can also extract the `v1/user-tasks` base path for user tasks into a constant in this test so that any future changes to the path are more easily adjustable.